modules/stats/drawdown/for_portfolio.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+
+
+def get_max_seen_drawdown_for_portfolio(capital_per_timestamp: dict):
+    max_seen_drawdown = {}
+
+    df = pd.DataFrame.from_dict(capital_per_timestamp, columns=['value'], orient='index')
+    df["drawdown"] = df["value"] / df["value"].cummax()
+
+    max_seen_drawdown["drawdown"] = df["drawdown"].min()
+    max_seen_drawdown["at"] = df["drawdown"].idxmin()
+    max_seen_drawdown["from"] = df.loc[:max_seen_drawdown["at"]].value.idxmax()
+    df_top_slice = df.loc[max_seen_drawdown["at"]:]
+    df_next_peak = df_top_slice.loc[df_top_slice["drawdown"] == 0]
```

Comment:
```suggestion
    df_next_peak = df_top_slice.loc[df_top_slice["drawdown"] == 1]
```
Shouldn't this be 1? As you reach the previous drawdown value and we use ratios?
I don't fully grasp how the top_slice and next_peak work, maybe we could also check at making better naming? 
df_top_slice > df_after_max_drawdown
df_next_peak > df_after_recovery

Reply:
Bug fixed (thanks for the catch!!), and variables renamed. closing convo!